intezer_sdk/_api.py

```diff
@@ -699,6 +702,11 @@ def get_alerts_by_alert_ids(self, alert_ids: List[str], environments: List[str]
         data = dict(alert_ids=alert_ids)
         if environments:
             data['environments'] = environments
+        
+        tenant_id = self._get_tenant_id()
+        if tenant_id:
```

Comment:
Since walrus is supported from python 3.8 on: you can do `if tenant_id := self._get_tenant_id()`